packages/react/src/textarea/Textarea.css.ts

```diff
@@ -0,0 +1,53 @@
+import { recipe } from "../vanilla-extract";
+
+export const textAreaBoxRecipe = recipe({
+  base: {
+    background: "transparent",
+    flexGrow: "1",
+    fontFamily: "sans",
+    padding: "8px",
+    selectors: {
+      "&:focus-visible": {
+        outline: "0",
+      },
+    },
+  },
+  variants: {},
+});
+
+export const parentBoxRecipe = recipe({
+  base: {
+    background: "white",
+    border: "1",
+    borderColor: "border.default",
+    color: "fg.default",
+    display: "flex",
+    flexDirection: "column",
+    fontSize: "md",
+    overflow: "auto",
+
+    selectors: {
+      "&:aria-invalid": {
+        borderColor: "border.error",
+      },
+      "&:data-disabled": {
+        backgroundColor: "bg.disabled",
+        borderColor: "border.secondary",
+        color: "fg.quaternary",
+        pointerEvents: "none",
+      },
+      "&:focus-within": {
+        outline: "2",
+        outlineColor: "brand.200",
```

Comment:
we can't refer to design tokens directly in recipe - it has to be through `theme` or via `sprinkles()`

```suggestion
        outlineColor: theme.colors["brand.200"],
```